src/core/token/ERC1155Core.sol

```diff
@@ -173,6 +174,10 @@ contract ERC1155Core is ERC1155, Core, Multicallable, EIP712 {
             selector: UpdateMetadataCallbackERC1155.updateMetadataERC1155.selector,
             mode: CallbackMode.REQUIRED
         });
+        supportedCallbackFunctions[8] = SupportedCallbackFunction({
+            selector: UpdateTokenIdCallbackERC1155.updateTokenIdERC1155.selector,
+            mode: CallbackMode.REQUIRED
```

Comment:
would it make sense to have it optional, and by default just return the token id passed in?

Reply:
Makes sense to me, updated